plugin-server/src/capabilities.ts

```diff
@@ -19,7 +19,7 @@ export function getPluginServerCapabilities(config: PluginsServerConfig): Plugin
                 processAsyncOnEventHandlers: true,
                 processAsyncWebhooksHandlers: true,
                 sessionRecordingBlobIngestion: true,
-                personOverrides: config.POE_DEFERRED_WRITES_ENABLED,
+                personOverrides: true,
```

Comment:
This isn't required yet/either for hobby installs, but no harm in just letting it run anyway.